src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/ReconciliationReportRepositoryTest.java

```diff
@@ -117,4 +135,76 @@ void should_return_empty_optional_if_report_does_not_exist() {
         // then
         assertThat(report).isEmpty();
     }
+
+    @Test
+    void should_not_find_anything_when_db_is_empty() {
```

Comment:
There are now 2 methods in this repository class that search for report.
One by ID, and the new one by date and account.

There's a test here already called `should_return_empty_optional_if_report_does_not_exist` (for the prior method).

It would be nice (perhaps on a future PR) to rename those tests so that it's clear what's tested.

Reply:
yes, saw it. left aside so to just deal with report itself